PRIVACY_AND_SECURITY.md

```diff
@@ -0,0 +1,82 @@
+# Answers to [Security and Privacy Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/)
+
+### 3.1 Does this specification deal with personally-identifiable information?
+
+No.
+
+
+### 3.2 Does this specification deal with high-value data?
+
+No.
+
+
+### 3.3 Does this specification introduce new state for an origin that persists across browsing sessions?
+
+No.
+
+
+### 3.4 Does this specification expose persistent, cross-origin state to the web?
+
+No.
+
+### 3.5 Does this specification expose any other data to an origin that it doesn’t currently have access to?
+
+Kind of. Origins will explicitly be told how they have been launched. Previously, they would have had to infer this based on requests (though this information **WAS** previously inferrible).
+
+### 3.6 Does this specification enable new script execution/loading mechanisms?
+
+No.
+
+
+### 3.7 Does this specification allow an origin access to a user’s location?
+
+No.
+
+
+### 3.8 Does this specification allow an origin access to sensors on a user’s device?
+
+No.
+
+
+### 3.9 Does this specification allow an origin access to aspects of a user’s local computing environment?
+
+No.
+
+
+### 3.10 Does this specification allow an origin access to other devices?
+
+No.
+
+
+### 3.11 Does this specification allow an origin some measure of control over a user agent’s native UI?
+
+No.
```

Comment:
Should this be 'yes' since the origin can control whether to focus an existing window or open a new one or show a notifications?

Reply:
Good point, I was thinking of this as adding new UI but that's probably an important thing to mention.